src/ui/public/directives/paginate.js

```diff
@@ -98,6 +98,10 @@ uiModules.get('kibana')
         }
       };
 
+      self.goToTop = function () {
```

Comment:
Could you convert this to a named function, to make it easier to follow the call stack?

```javascript
self. goToTop = function goToTop() {
  // ...
};
```